optaplanner-wb-webapp/src/main/resources/org/optaplanner/workbench/client/resources/i18n/AppConstants.properties

```diff
@@ -17,7 +16,6 @@ HomeProducer.ProjectAuthoring=Project Authoring
 HomeProducer.ArtifactRepository=Artifact Repository
 HomeProducer.Administration=Administration
 HomeProducer.DeployYourArtifacts=Deploy your Artifacts
-
```

Comment:
Are you sure it's a good thing that it removed these blank lines in .properties files?